_episodes_rmd/.ignore-05-databases.Rmd

```diff
@@ -28,9 +28,14 @@ source("../bin/download_data.R")
 
 ## Introduction
 
-A common problem with R in that all operations are conducted in-memory and thus
-the amount of data you can work with is limited by available memory. So far, we have used small datasets
-that can easily fit into your computer's memory. But what about datasets that are too large for your
+A common problem with R in that all operations are conducted in-memory. Here, "memory" means a computer 
+drive called Random Access Memory (RAM for short) which is phsically installed on your computer. RAM 
+allows for data to be read in in nearly the same amount of time regardless of its physical location
```

Comment:
duplicate "in in"

Reply:
I agree that this sentence was awkward, my bad. I was attempting to say that the data is being "read in" / imported _in_ nearly the same amount of time.